src/js/components/viewer/models/audio-media-object.jsx

```diff
@@ -200,43 +216,51 @@ var AudioMediaObject = React.createClass({
     },
 
     destoryAudioPlayer: function() {
+
+        // APEP Ensure we clean up the use of the Tween library
+        if(this.state.transitionToDoneAudioTween) {
+            this.state.transitionToDoneAudioTween.stop();
+        }
+
         // APEP Ensure the Audio5 component cleans up
         if(this.state.player) {
             this.state.player.pause();
             this.state.player.destroy();
             this.state.player.off("timeupdate", this.audioPlayerTimeUpdate);
-            this.state.player = null;
```

Comment:
I removed this for two reasons:

1. State update patterns during unmounting is not valid.
2. This is not a valid way to change state for a react component.